common/test/acceptance/tests/studio/test_studio_container.py

```diff
@@ -1056,3 +1057,53 @@ def test_display_name_default(self):
         title_on_unit_page = test_block.name
         container = test_block.go_to_container()
         self.assertEqual(container.name, title_on_unit_page)
+
+
+class ProblemCategoryTabsTest(ContainerBase):
+    """
+    Test to verify tabs in problem category.
+    """
+    def setUp(self, is_staff=True):
+        super(ProblemCategoryTabsTest, self).setUp(is_staff=is_staff)
+
+    def populate_course_fixture(self, course_fixture):
+        """
+        Sets up course structure.
+        """
+        course_fixture.add_children(
+            XBlockFixtureDesc('chapter', 'Test Section').add_children(
+                XBlockFixtureDesc('sequential', 'Test Subsection').add_children(
+                    XBlockFixtureDesc('vertical', 'Test Unit')
+                )
+            )
+        )
+
+    def test_correct_tabs_present(self):
+        """
+        Scenario: Verify that correct tabs are present in problem category.
+        """
+        self.go_to_unit_page()
+        page = ContainerPage(self.browser, None)
+        self.assertTrue('Common Problems with Hints and Feedback' not in page.get_category_tab_names('problem'))
```

Comment:
Can you check that the other tabs are present? In fact, an exact list match would be nice.